mifospay/src/main/res/values/strings.xml

```diff
@@ -249,5 +249,6 @@
     <string name="hide_balance">Hide Balance</string>
     <string name="no_bank_account">No bank found</string>
     <string name="reset_amount">Reset Amount Successful</string>
+    <string name="cancel">Cancel</string>
```

Comment:
Delete this "cancel" string, we already have it use that one only. Because of this, your build is failing.